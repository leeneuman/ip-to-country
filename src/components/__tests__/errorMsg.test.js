import { render, screen, cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';
import ErrorMsg from './../errorMsg';

afterEach(cleanup);

test('should render error msg component', () => {
	render(<ErrorMsg error="test error msg" />);
	const errorElement = screen.getByTestId('error-msg');
	expect(errorElement).toBeInTheDocument();
	expect(errorElement).toHaveTextContent('test error msg');
});

test('matches snapshot', () => {
	const tree = renderer.create(<ErrorMsg error="test error msg" />).toJSON();
	expect(tree).toMatchSnapshot();
});
