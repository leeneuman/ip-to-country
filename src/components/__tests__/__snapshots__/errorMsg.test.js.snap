// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="mt-1 text-danger"
  data-testid="error-msg"
>
  test error msg
</div>
`;
